mucommander-core/src/main/java/com/mucommander/ui/main/WindowManager.java

```diff
@@ -181,25 +199,27 @@ public static void tryRefreshCurrentFolders() {
     /**
      * Creates a new MainFrame and makes it visible on the screen, on top of any other frames.
      *
-     * @param leftFolders initial paths for the left frame.
-     * @param rightFolders initial paths for the right frame.
+     * @param mainFrameBuilder the mainFrame builder
      * @return the newly created MainFrame.
      */
     public static synchronized void createNewMainFrame(MainFrameBuilder mainFrameBuilder) {
+        LOGGER.error("creating new main frame...");
```

Comment:
remove the 'error' logs from here as well?

Reply:
-> debug